Examples/OneSignalDemo/app/build.gradle

```diff
@@ -0,0 +1,44 @@
+//apply plugin: 'com.onesignal.androidsdk.onesignal-gradle-plugin'
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 28
+    defaultConfig {
+        applicationId "com.onesignal.sdktest"
+        minSdkVersion 21
+        targetSdkVersion 28
+        versionCode 1
+        versionName "1.0"
+        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+    }
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+    }
+}
+
+dependencies {
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
+    implementation 'com.android.support:design:28.0.0'
+    implementation 'com.android.support:cardview-v7:28.0.0'
+    implementation 'com.android.support:appcompat-v7:28.0.0'
+    implementation 'com.android.support:support-v4:28.0.0'
+    implementation 'com.android.support:support-vector-drawable:28.0.0'
+    implementation 'com.android.support.constraint:constraint-layout:1.1.3'
+
+//    implementation 'com.google.firebase:firebase-core:16.0.9'
+
+    implementation(project(':onesignal'))
+
+//    implementation 'com.onesignal:OneSignal:3.12.5'
```

Comment:
Uncomment, example app should be using the production SDK

Reply:
One is for testing and one is for local SDK.

Public release is not going to build until manifest placeholders are gone gone

Reply:
got it